Gems/OpenXRVk/Code/Source/OpenXRVkDevice.cpp

```diff
@@ -170,7 +170,9 @@ namespace OpenXRVk
 
     bool Device::AcquireSwapChainImageInternal(AZ::u32 viewIndex, XR::SwapChain* baseSwapChain)
     {
+#if !defined(RELEASE)
         SwapChain* swapChain = static_cast<SwapChain*>(baseSwapChain);
```

Comment:
This is a little messy. Can we just move this cast into the assert itself. Then we wont need this ifdef

Reply:
done